src/bin/errors.js

```diff
@@ -8,4 +8,9 @@ module.exports = function registerErrorHandler(program) {
     console.error(`Invalid command: ${program.args.join(' ')}\nSee --help for a list of available commands.`)
     process.exit(1)
   })
+
+  program.parse(process.argv)
```

Comment:
We shouldn't be parsing args here, we are already doing that in `bin/zos-cli.js`. Thus we should eval if no args were given there.